File: Server/Station.js
const mongoose = require('mongoose')

const StationSchema = new mongoose.Schema({
    location: { type: String, required: true },
    name: { type: String, required: true },
    longitude: { type: Number, required: true },
    latitude: { type: Number, required: true },
    camera : {
        state: { type: Boolean, required: true },
        time_stamp: { type: String },
    },
    drone : {
        state: { type: Boolean, required: true },
        battery: { type: Number, required: true },
        time_stamp: { type: String },
    },
    raspberry : {
        state: { type: Boolean, required: true },
        time_stamp: { type: String },
    },
    warning: { type: Boolean, required: true }
}, { versionKey: false });

const Station = mongoose.model('Station', StationSchema)
module.exports = Station;

File: emqx/Station.py
class Station:
    def __init__(self, location, name, longitude, latitude, camera, drone, raspberry, warning): 
        self.location = location
        self.name = name
        self.longitude = longitude
        self.latitude = latitude
        self.camera = camera
        self.drone = drone
        self.raspberry = raspberry
        self.warning = warning
    
    def to_json(self):
        return {
            "location": self.location,
            "name": self.name,
            "longitude": self.longitude,
            "latitude": self.latitude,
            "camera": self.camera.to_json(),
            "drone": self.drone.to_json(),
            "raspberry": self.raspberry.to_json(),
            "warning": self.warning
        }